torsiondrive/tests/test_reproduce_examples.py

```diff
@@ -18,7 +18,7 @@ def example_path(tmpdir_factory):
     # tmpdir_factory is a pytest built-in fixture that has "session" scope
     tmpdir = tmpdir_factory.mktemp('torsiondrive_test_tmp')
     tmpdir.chdir()
-    example_version = '0.9.5'
+    example_version = '0.9.5.1'
```

Comment:
The new example release have updated results for `api_example` with sorted output.